service/src/main/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentService.java

```diff
@@ -108,15 +110,20 @@ private CreditAccountPaymentRequest getCreditAccountPaymentRequest(Long caseId,
             .build();
     }
 
-    private void callPaymentsApi(CreditAccountPaymentRequest creditAccountPaymentRequest) {
+    @Retryable(value = PaymentRetryException.class)
+    //had to make this public for retry to work
```

Comment:
if there is test that verify this, comment is not needed

Reply:
planning to add : @Retryable(value = {RetryablePaymentException.class}, label = "payment api call")